From cf84bf3ef505059d42184b22cf38f89336bec43d Mon Sep 17 00:00:00 2001
From: Gergely Imreh <imrehg@gmail.com>
Date: Mon, 22 Jun 2015 07:51:17 +0000
Subject: [PATCH 1/1] remove hard-wired ncursesw include path

Don't assume that the ncursesw headers are in ../usr/include/ncursesw/..,
and pkg-config finds the correct include path anyways.

Signed-off-by: Gergely Imreh <imrehg@gmail.com>
---
 pinentry/pinentry-curses.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/pinentry/pinentry-curses.c b/pinentry/pinentry-curses.c
index 235435a..bdcd0f4 100644
--- a/pinentry/pinentry-curses.c
+++ b/pinentry/pinentry-curses.c
@@ -22,11 +22,7 @@
 #include <config.h>
 #endif
 #include <assert.h>
-#ifdef HAVE_NCURSESW
-#include <ncursesw/curses.h>
-#else
 #include <curses.h>
-#endif
 #include <signal.h>
 #include <fcntl.h>
 #include <unistd.h>
-- 
1.9.1

